src/modules/previewpane/powerpreview/settings.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <pch.h>
+#include <string>
+#include "resource.h"
+#include <settings_objects.h>
+#include "registry_wrapper_interface.h"
+
+namespace PowerPreviewSettings
+{
+	// PowerToy Winodws Explore File Preview Settings.
+	class FileExplorerPreviewSettings
```

Comment:
why do we use registry for settings instead of json file?

Reply:
We are using `json` file to store the settings. The case where we are updating the registry is to provide on/off property for preview handlers by deleting the registry value added by the installer in `HKCU\Software\Microsoft\Windows\CurrentVersion\PreviewHandlers`.